webroot/.htaccess

```diff
@@ -1,5 +1,7 @@
 <IfModule mod_rewrite.c>
     RewriteEngine On
-    RewriteCond %{REQUEST_FILENAME} !-f
-    RewriteRule ^ index.php [L]
+    # RewriteBase / may be required by some hosts
+    #RewriteBase   /
```

Comment:
`RewriteBase   /`
Broke my app  in local :)

Reply:
Really? Never broke mine neither in cakephp2 nor 3.

Reply:
it brokes my app also

Reply:
I think this is related https://github.com/joomla/joomla-cms/blob/staging/htaccess.txt#L58-L64